wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -0,0 +1,724 @@
+- [Understanding the GitHub's workflow](https://help.github.com/articles/what-is-a-good-git-workflow/ "GitHub Help on GitHub workflow").
+  - A simplified visual explanation can be found [here](https://guides.github.com/introduction/flow/).
+- Able to write in English, preferably British English for consistency.
+  - Everything not related to the page content itself (commit messages, pull requests, filenames, etc) **MUST** be written in English.
+
+Optionally, if you plan to make changes locally:
+
+- A text editor with Markdown syntax highlighting.
+  - We either use Microsoft's [Visual Studio Code](https://code.visualstudio.com "Visual Studio Code") or GitHub's [Atom](https://atom.io "Atom").
+  - Any text editor will suffice if you do not need the syntax highlighting.
+- A Markdown previewer with **GFM** support.
+  - We use [Shiba](https://github.com/rhysd/Shiba "Shiba"), or Visual Studio Code's/Atom's built-in GFM previewers.
+- Basic knowledge in VCS (Version control system), particularly [Git](https://git-scm.com/ "Git").
+
+## Forking
+
+To be able to make changes to the repository, you must first create your own copy of the repository.
+This is known as "forking".
+
+Go to the [osu!wiki repository](https://github.com/ppy/osu-wiki).
+
+You should see something like this.
+
+![osu-wiki Repository Snapshot](Contribute_wikirepo.jpg "Snapshot of the osu!Wiki Repository")
+
+Click on the "Fork" button, located at the top-right of the page.
+This should redirect you to the location for the fork.
+
+![Fork prompt dialogue](Contribute_forkprompt.jpg "Fork prompt dialogue")
+
+Now click at your profile and GitHub would redirect you to where the fork is.
+
+Finally, you should see something like this.
+
+![Fork Snapshot](Contribute_ownrepo.jpg "Fork Snapshot")
+
+Congratulations!
+You have made a copy of the osu!wiki repository!
+
+You can now proceed to make changes.
+
+## Making Changes
+
+### Through GitHub's Web Editor
+
+#### Accessing the correct repository and branch.
+
+If you don't want the hassle of cloning your fork into your local machine, GitHub offers a web editor for editing through your browser.
+
+First, visit your own forked repo (you can rename the ``ppy`` part in ``https://github.com/ppy/osu-wiki`` with your own GitHub's name for quicker access, or click on the "Fork" button in ``ppy/osu-wiki``).
+
+You should be in the ``master`` branch by default.
+
+#### Branching
+
+Frankly speaking, you can do all your changes in the ``master`` branch, but it is a _bad practice in the long run_.
+
+This is where branching come into play.
+By branching, you keep your original ``master`` branch clean.
+In the event of major slip-ups, you can cut off the branch and start a new branch copy based on the clean branch.
+
+![Branch dropdown list](Contribute_webbranch.jpg "Branch dropdown list")
```

Comment:
If you need to obscure images... why not use the devtools to modify the DOM or something? If you need help with obscuring images, let me know.